Meadowlark-js/backends/meadowlark-search-backend/src/repository/ClientSearch.ts

```diff
@@ -0,0 +1,41 @@
+import { Client as OpenSearchClient } from '@opensearch-project/opensearch';
+import { Client as ElasticClient } from '@elastic/elasticsearch';
+import { /* Config, */ Logger } from '@edfi/meadowlark-utilities';
+import { BasicAuth } from '@opensearch-project/opensearch/lib/pool';
+
+export type ClientSearch = OpenSearchClient | ElasticClient;
+
+const moduleName = 'opensearch.repository.Db';
+
+function getSearchAPI() {
+  // return Config.get<string>('SEARCH_API');
```

Comment:
And using `SEARCH_PROVIDER` as the string here?

Reply:
Sure, sounds good. Here is where I have some problems. If I uncomment this line, I get an error saying that the variable does not exist.